src/contracts/Interfaces.sol

```diff
@@ -255,3 +255,8 @@ interface IStETHWithdrawal {
     function getWithdrawalRequests(address _owner) external view returns (uint256[] memory requestsIds);
     function getLastRequestId() external view returns (uint256);
 }
+
+interface ILidoARM {
```

Comment:
There's a ILiquidityProviderARM interface that already has deposit. We can extend IERC20 so we also get transfer.